docs/administration/robot-accounts/_index.md

```diff
@@ -5,39 +5,27 @@ weight: 40
 
 Harbor v2.2 introduces the capability for administrators to create system robot accounts you can use you run automated actions in your Harbor instances. System robot accounts allow you to use a robot account to perform maintenance or repeated tasks across all or a subset of projects in your Harbor instance.
 
-For each system robot account you are able to assign the following permissions,
-
-* List Repository
-* Pull Repository
-* Push Repository
-* Delete Repository
-* Read Artifact
-* List Artifact
-* Delete Artifact
-* Create Artifact Label
-* Delete Artifact Label
-* Create Tag
-* Delete Tag
-* List Tag
-* Create Scan
-* Stop Scan
-
-Depending on your needs you can assign a combination of these permissions to a system robot account to perform your desired tasks through the OCI client or Harbor API. Robot Accounts cannot log in to the Harbor interface.
-
-You are also able create project scope robot account that only have access to a single project. Read more about [project robot accounts](../../working-with-projects/project-configuration/create-robot-accounts/).
+For each system robot account you are able to assign the system permissions and specify the projects it covers. And for each of the specified projects you are able to assign the project permissions
+
+You can refer to the [**Permission References**](#permission-references) to assign a combination of these permissions to a system robot account to perform your desired tasks through the OCI client or Harbor API. Robot Accounts cannot log in to the Harbor interface.
+
+You are also able to create project scope robot account that only have access to a single project. Read more about [project robot accounts](../../working-with-projects/project-configuration/create-robot-accounts/).
 
 ## View System Robot Accounts
 
 1. Log into your harbor instance as an administrator.
```

Comment:
Harbor is a name, so first latter in capital: > Log into your Harbor instance as an administrator.